zcbor/zcbor.py

```diff
@@ -756,145 +759,146 @@ def convert_to_key(self):
         convert_val.min_qty = 1
         convert_val.base_name = None
 
-    # Parse from the beginning of instr (string) until a full element has been parsed. self will
-    # become that element. This function is recursive, so if a nested element
-    # ("MAP"/"LIST"/"UNION"/"GROUP") is encountered, this function will create new instances and add
-    # them to self.value as a list. Likewise, if a key or cbor definition is encountered, a new
-    # element will be created and assigned to self.key or self.cbor. When new elements are created,
-    # get_value() is called on those elements, via parse().
-    def get_value(self, instr):
+    # A dict with lists of regexes and their corresponding handlers.
+    # This is a dict in case multiple inheritors of CddlParser are used at once, in which case
+    # they may have slightly different handlers.
+    cddl_regexes = dict()
+
+    # Initialize the cddl_regexes variable.
+    def initialize_regex_list(self):
```

Comment:
nit: What about renaming the function to `cddl_regexes_init()`, or similar name that would include `cddl_regexes`. That would enable to remove the comment a line above.

Reply:
Good idea, done.